clio-client/src/main/scala/org/broadinstitute/clio/client/util/IoUtil.scala

```diff
@@ -111,6 +116,28 @@ trait IoUtil {
   def listGoogleObjects(path: URI): Seq[String] = {
     gsUtil.ls(path.toString)
   }
+
+  /**
+    * Check if a path points to an object in GCS.
+    *
+    * If so, return the object's size and md5 hash (if present).
+    */
+  def getGoogleObjectInfo(path: URI): Option[(Long, Option[Symbol])] = {
```

Comment:
Hrm, not sure about this returning an `Option[_]` any more. When I use it in the backfill I end up doing a `.getOrElse { throw new RuntimeException(...) }` on the result every time. Thoughts from the reviewer?

Reply:
would someone use this method to test existence? If so I think the Option makes sense. If not I think its fine to remove it.